dashboard/test/ui/features/plc/courseUnitNavigation.feature

```diff
@@ -0,0 +1,30 @@
+@dashboard_db_access
+Feature: Basic navigation for PLC stuff
+
+Background:
+  Given I am a teacher
+  And I am enrolled in a plc course
+  Given I am on "http://studio.code.org/course/csp-support"
```

Comment:
We should create an `/s/alltheplcthings` script specifically for UI tests so content changes to `/s/csp-support` don't break our UI tests.